modules/nodes-cluster-limit-ranges-about.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * nodes/cluster/limit-ranges.adoc
+
+[id="nodes-cluster-limit-ranges-about_{context}"]
+= About limit ranges
+
+A limit range, defined by a `*LimitRange*` object, restricts resource 
+consumption in a project. In the project you can set specific resource 
+limits for a Pod, container, image, image stream, or 
+persistent volume claim (PVC).
+
+All requests to create and modify resources are evaluated against each
+`*LimitRange*` object in the project. If the resource violates any of the
+enumerated constraints, the resource is rejected. 
+
+ifdef::openshift-online[]
+[IMPORTANT]
+====
+For {product-title} Pro, the maximum Pod memory is 3Gi. The minimum Pod or
```

Comment:
Do we even have Starter anymore? This needs to be confirmed.

Reply:
Yep.

Reply:
Values did not change per @tiwillia in private Slack: https://coreos.slack.com/archives/D0180S7QVJN/p1596131179002900